components/tag/index.css

```diff
@@ -26,8 +26,12 @@ governing permissions and limitations under the License.
   --spectrum-tag-label-line-height: var(--spectrum-line-height-100);
   --spectrum-tag-label-font-weight: var(--spectrum-regular-font-weight);
 
-  /* selected content */
-  --spectrum-tag-content-color-selected: var(--spectrum-white);
+  /* selected */
+  --spectrum-tag-content-color-selected: var(--spectrum-gray-50);
```

Comment:
Should we add a note or TODO comment suggesting that mapping to global variables is a temporary solution until the design tokens for this are available?

Reply:
in this case this component has already been migrated and the XD file listed `--spectrum-gray-50` as the token